webapp/client/src/components/ShareIcons.js

```diff
@@ -0,0 +1,95 @@
+import React from "react";
+import { SocialIcon } from "react-social-icons";
+import { isMobile } from "react-device-detect";
+import styled from "styled-components";
+import PropTypes from "prop-types";
+
+const FacebookIcon = styled(SocialIcon)`
+  // This is needed for the hover effect on the background when going over the icon
+  .social-svg-icon:hover + .social-svg-mask {
+    fill: #2f4779 !important;
+  }
+
+  .social-svg-mask:hover {
+    fill: #2f4779 !important;
+  }
+`;
+
+const WhatsappIcon = styled(SocialIcon)`
+  .social-svg-icon:hover + .social-svg-mask {
+    fill: #25c05f !important;
+  }
+
+  .social-svg-mask:hover {
+    fill: #25c05f !important;
+  }
+`;
+
+const EmailIcon = styled(SocialIcon)`
+  .social-svg-icon:hover + .social-svg-mask {
+    fill: #6b6b6d !important;
+  }
+
+  .social-svg-mask:hover {
+    fill: #6b6b6d !important;
+  }
+`;
+
+function sendGoal(plausibleDomain, iconName, sourcePage) {
+  if (plausibleDomain !== null) {
+    window.plausible(`${iconName} icon clicked`, {
+      props: { method: sourcePage },
+    });
+  }
+}
```

Comment:
As we are going to need this functionality more often in the future, I am wondering if we want to directly take that out of here and make it more general (I am fine with both though)

Reply:
You mean like extracting the adding of a plausible goal with variable name and props? That would mean just creating a simple and small "global" function like:

`function sendGoal(plausibleDomain, name, props) {
  if (plausibleDomain !== null) {
    window.plausible(name, props);
  }
}`
?

Reply:
That's exactly what I was thinking of

Reply:
As discussed in our daily today, i will create the global function in the main branch directly, so that other current branches can use it already